crates/cli/src/lib.rs

```diff
@@ -406,17 +406,22 @@ pub fn test(matches: &ArgMatches, triple: Triple) -> io::Result<i32> {
 
     // Spawn the root task
     if !path.exists() {
-        let path_string = path.to_string_lossy();
+        let current_dir = env::current_dir().unwrap();
+        let expected_file_path = current_dir.join(filename);
+
+        let current_dir_string = current_dir.to_string_lossy();
+        let expected_file_path_string = expected_file_path.to_string_lossy();
```

Comment:
would https://doc.rust-lang.org/std/path/struct.Path.html#method.display work here? I think that would prevent allocations in some cases

Reply:
`display` appears to work just as well; I was using `to_string_lossy` just because it was already there, should I change it?

Reply:
yes let's try `display` instead.